project-ideas_2023/README.md

```diff
@@ -6,18 +6,16 @@
 
 ## Project Ideas
 
-### 1. aRMT Protocol Generator and Questionnaire Tester
+### 1. aRMT Questionnaire and Protocol Generator
 
 **Overview**: The [RADAR-base Active Mobile app](https://github.com/RADAR-base/RADAR-Questionnaire) (also known as Questionnaire app) is an app used to deliver questionnaires to participants in a study. The app takes in schedule protocols from a protocol repository ([RADAR-aRMT-Protocols](https://github.com/RADAR-base/RADAR-aRMT-protocols)) and questionnaires from a questionnaire definitions repository ([RADAR-REDCap-aRMT-Definitions](https://github.com/RADAR-base/RADAR-REDCap-aRMT-Definitions)).
 
 **Current Status**: At the moment, the questionnaire and protocol JSON files are manually written or generated by external tools. This process is error-prone and less user-friendly.
- 
-**Goals:** 
-The goal of this project is to implement a UI which can be used to generate the JSON file based on user input and store this generated JSON file to be used by the app. The app should be able to adapt the input type user has chosen and render a sample question that would appear on the app (See wireframe below). We should be able to reuse the components created as part of the questionnaire app to build the live rendering.
 
-Questionnaire creation should also support branching logic which is supported by the app.
+### 1A. aRMT Questionnaire Generator
 
-Second stage of this project is to apply the same to generate protocols. Live rendering is not necessary for this. Optionally, if time permits this can be further extend to validate the protocol by generating schedule used on the enrolment-time of a participant.
+**Goals:** 
+The goal of this project is to implement a UI which can be used to generate the questionnaire definition JSON file based on user input and store this generated JSON file to be used by the app. The app should be able to adapt the input type user has chosen and render a sample question that would appear on the app (See wireframe below). We should be able to reuse the components created as part of the questionnaire app to build the live rendering. Questionnaire creation should also support branching logic which is supported by the app.
```

Comment:
Maybe link to an example definitions file would be good to include. Similar for protocol file.